src/apps/file/router.py

```diff
@@ -0,0 +1,8 @@
+from fastapi.routing import APIRouter
+
+from apps.file.api.file import download, upload
+
+router = APIRouter(prefix="/file", tags=["File"])
+
+router.post("/upload")(upload)
+router.post("/download")(download)
```

Comment:
What kind of files are you going to upload/download via those endpoints?
Are you going to add some code documentations?